shared/middleware/helpers/storage_apps.rb

```diff
@@ -138,6 +139,15 @@ def get_abuse(channel_id)
     row[:abuse_score]
   end
 
+  def get_skip_content_moderation_value(channel_id)
```

Comment:
In https://github.com/code-dot-org/code-dot-org/pull/23406 you settled on `disable-auto-moderation` as the consistent language for this feature.  Our server code should use the same language, unless there's a particular reason it's different.

Reply:
Idiomatic ruby nit: Name methods that return a boolean value as a query `def auto_moderation_disabled?(channel_id)`